adapter_pipelines/10x/adapter.wdl

```diff
@@ -10,7 +10,7 @@ task GetInputs {
 
   command <<<
     python <<CODE
-    from pipeline_tools import utils
+    from pipeline_tools import dcp_utils
```

Comment:
Do you also need to update the docker container so that it has the `dcp_utils` file?

Reply:
Yes, I did. Now it is `humancellatlas/pipeline-tools:0.1.5`.

Reply:
Great! It should also be updated in this task too, which is currently still `humancellatlas/pipeline-tools:0.1.4`: https://github.com/HumanCellAtlas/pipeline-tools/blob/53220670659f173571519598745babfae793b21c/adapter_pipelines/10x/adapter.wdl#L186

Reply:
Thanks for this catch! I planned to update all the docker tags once I make the dcp-wide integration test works again. Also, I need to update some of the files according to the v4.6.0 metadata schema update.